lib/engine/round/base.rb

```diff
@@ -235,6 +235,21 @@ def can_gain?(share, entity)
         corporation.holding_ok?(entity, share.percent) &&
         (!corporation.counts_for_limit || entity.num_certs < @game.cert_limit)
       end
+
+      def liquid_bundle?(bundle, entity)
+        return false unless @game.turn > 1
```

Comment:
this should be factored out from round/stock.rb#20
can_sell is defined there already